finegrain/src/finegrain/__init__.py

```diff
@@ -248,15 +272,25 @@ def __init__(
         self.verify = verify
         self.default_timeout = default_timeout
 
-        self.token = None
         self.logger = logger
+        self._sse_source = ResilientEventSource(
+            url=self.get_sub_url,
+            ping_interval=self.get_ping_interval,
+            verify=self.verify,
+        )
+        self.reset()
 
+    def reset(self) -> None:
```

Comment:
Should be ignored in most vanilla use cases? To put it differently: when should this be called explicitly (during the lifecycle of the EditorAPIContext)?

Reply:
Typically not (can be ignored).